### DISCUSSION TITLE
Control panel 404 when Installing Statamic 4 in Laravel 10 app

### DISCUSSION THREAD

**Discussion by tom-sk** [statamic/cms#8382] [Troubleshooting]
Hi,

I have just tried installing Statamic 4 into a new Laravel 10 app and /cp is returning 404.

I am using MAMP and PHP 8.2

Steps to reproduce:
- composer create-project laravel/laravel basic-laravel
- follow steps listed here: https://statamic.dev/installing/laravel
- Add users to DB and run migrations
- 404

**Comment by tom-sk** [comment]
After installing a few different projects doing the exact same process it stopped 404ing, so not suer why this was happening?

**Reply by jackmcdade** [reply]
So is it till an issue or not? If you can recreate the 404 problem reliably, I'm sure we solve it.

**Reply by tom-sk** [reply]
I can no longer recreate, will close the issue
